bench/scrypt.cc

```
@@ -0,0 +1,46 @@
+  for (auto _ : state) {
+    uint8_t out[64];
+    if (!EVP_PBE_scrypt(kPassword, sizeof(kPassword) - 1, kSalt,
+                        sizeof(kSalt) - 1, N, r, p, 0 /* max_mem */, out,
+                        sizeof(out))) {
+      state.SkipWithError("scrypt failed.");
+      return;
+    }
+  }
+}
+
+BSSL_BENCH_LAZY_REGISTER() {
```

Clang (14.0.6)

[warning] variable 'register_token39' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)